include/fiction/io/print_layout.hpp

```diff
@@ -307,6 +316,86 @@ void print_cell_level_layout(std::ostream& os, const Lyt& layout, const bool io_
     // flush stream
     os << std::endl;
 }
+/**
+ * Writes a simplified 2D representation of a SiDB charge layout to an output stream.
```

Comment:
```suggestion
 * Writes a simplified 2D representation of an SiDB charge layout to an output stream.
```

Reply:
Fixed this in the next commit, though I'm waiting for the tests this time.